### DISCUSSION TITLE
Open using specific profile

### DISCUSSION THREAD

**Discussion by mikel** [will-stone/browserosaurus#495] [Ideas]
Hey there Will,

Loving browserosaurus, thank you, I use it so many times per day its become a close and trusted friend :)

Question, I would _love_ the capability of being able to treat the profiles within specific browsers as different browsers.  For example, within Chrome I have 4 profiles (for different things), would it be possible to be able to select these as different browsers in the browserosaurus select box?  I don't even know if this is possible technically...

Currently, browserosaurus opens the last profile, which just means I have to cancel the browserosaurus pop up, then open chrome, then switch profile, then click the link again then click chrome... bunch of extra clicks.

Ideal scene would be that each profile is detected as a separate browser with the option to name them.

What do you think?

**Comment by will-stone** [comment]
Hi Mikel,

Thank you for the kind words and using Browserosaurus. This request has [come up several times](https://github.com/will-stone/browserosaurus/issues?q=is%3Aissue+profile+is%3Aclosed) throughout the lifetime of the app but I'm sorry, it is really not something I'm interested in supporting as I don't use Chrome's profiles.

Thanks,

Will.

**Comment by mikel** [comment]
Hey there @will-stone understood... I just read through one of the feature requests and I can see that half the problem was google not providing the capability of doing it simply from the command line.

In any case, thanks for (1) your software, (2) your rapid reply!

I'll continue to support you and your project for as long as you keep it working.  Do well :). And if in the future chrome release a simple command line tool, I reserve the right to offer to pay you to implement it :)

Do well!

**Comment by mrsauravsahu** [comment]
Hey everyone! I recently discovered browserosaurus and loving it so far. I use profiles extensively and would love to implement this. I tried out various commands to launch a profile from the command line and it is true, the `--args` to the `open` command doesn't work right.

Need to do some more spike on this, but will definitely update this thread if I get it working.

**Reply by kristianatdoshii** [reply]
I've been using this command to open a new window, but I may explore how it can be done by Browserosaurus

`/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --args --profile-directory="Profile Name" --new-window`

**Reply by kristianatdoshii** [reply]
Working out the profile name structure is the hard part..
